registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/java/io/mosip/registration/processor/manual/verification/service/impl/ManualVerificationServiceImpl.java

```diff
@@ -622,11 +748,61 @@ public String getFilename() {
 
 		return matchedEntities;
 	}
+	
+	private ManualAdjudicationRequestDTO prepareManualAdjudicationRequestV1(MessageDTO messageDTO, List<ManualVerificationEntity> mve) throws Exception {
```

Comment:
i dont see anywhere this method is getting called? am i missing something here?

Reply:
I have added a flag to call either old or new method of 'prepareManualAdjudicationRequest' in registration-processor-mz.properties